test/assertions_test.rb

```diff
@@ -124,7 +124,7 @@ def test_assert_statsd_call
       end
     end
 
-    assert_assertion_triggered do
+    assert_no_assertion_triggered do
```

Comment:
This test change is based on the other change in this commit noted above, i.e. the check for tags is less strict that it used to be in that it'll validate that expected tags are a subset of actual tags for each matching metric.